service-worker/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
May not be relevant with babel, but this is only used in service-worker/index.ts I assume? Could we just hardcode these into that file?

Reply:
This tsconfig is for the actual service-worker/index.ts file. But it imports the root tsconfig.json for runtime compilation of the other sources.